src/object-marker.scss

```diff
@@ -0,0 +1,93 @@
+@import "./new-settings";
+@import "./mixins";
+
+/*!
+.fd-object-marker
+*/
+$block: #{$fd-namespace}-object-marker;
+
+.#{$block} {
+  // ICON VARIABLES
+  $fd-object-marker-icon-size: 1rem !default;
+  $fd-object-marker-color: var(--sapContent_MarkerIconColor) !default;
+  $fd-object-marker-text-color: var(--sapContent_LabelColor) !default;
+  $fd-object-marker-padding-left: 0.5rem !default;
+  $fd-object-marker-padding-right: 0.25rem !default;
+  // TEXT VARIABLES
+  $fd-object-marker-font-size: var(--sapFontSize) !default;
+
+  @include fd-reset();
+
+  font-size: $fd-object-marker-font-size;
```

Comment:
Let's remove ` font-size: $fd-object-marker-font-size;`. It's already in `fd-reset()` mixin. You can also remove this variable.
![image](https://user-images.githubusercontent.com/26483208/87645802-4d828f80-c74e-11ea-84c9-cdbbebb8b87c.png)

You can see that font-size is duplicated